vite.config.ts

```diff
@@ -10,12 +10,17 @@ if (fs.existsSync(".env.local")) {
   dotenv.config({ path: ".env.local" });
 }
 
+// Provide the SCSS settings from the _settings.scss file.
+// SCSS in react-components can reference variables like customized $breakpoint-large and should use our settings.
+const scssSettings = fs.readFileSync("src/sass/_settings.scss", "utf-8").trim();
```

Comment:
May be worth adding a comment in `_settings.scss` file as well, to make devs aware that this file is injected via Vite config, and warn them not to excessively edit it (outside of redefining Vanilla variable values).